src/Discovery/PathFinderCore.php

```diff
@@ -29,7 +29,7 @@ public function find($seed) {
     // up and up and up until we reach the Drupal root.
     do {
       if ($this->isDrupalRoot($directory)) {
-        return $this->cleanDirPath($directory->getPathName()) . $this->path;
+        return dirname(realpath($this->cleanDirPath($directory->getPathName())));
```

Comment:
:one: What's the purpose of `dirname(realpath(`.

:two: We need to add the path.